modules/broker-auth/src/main/java/io/ballerina/messaging/broker/auth/authorization/authorizer/empty/EmptyAuthResourceStore.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package io.ballerina.messaging.broker.auth.authorization.authorizer.empty;
+
+import io.ballerina.messaging.broker.auth.authorization.AuthResourceStore;
+import io.ballerina.messaging.broker.auth.authorization.authorizer.rdbms.resource.AuthResource;
+import io.ballerina.messaging.broker.auth.exception.BrokerAuthDuplicateException;
+import io.ballerina.messaging.broker.auth.exception.BrokerAuthException;
+import io.ballerina.messaging.broker.auth.exception.BrokerAuthNotFoundException;
+import io.ballerina.messaging.broker.auth.exception.BrokerAuthServerException;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Defines empty @{@link AuthResourceStore} when authorization is disabled.
+ */
+class EmptyAuthResourceStore implements AuthResourceStore {
+
+    @Override
+    public boolean authorize(String resourceType, String resourceName, String action, String userId,
+                             Set<String> userGroups) throws BrokerAuthServerException, BrokerAuthNotFoundException {
```

Comment:
Shall we remove redundant exception throws?

Reply:
Fixed in 384b733